influxdb2_operational_monitoring/influxdb2_cardinality_now.yml

```diff
@@ -1,36 +1,60 @@
 apiVersion: influxdata.com/v2alpha1
 kind: Label
 metadata:
-    name: kind-spence-53b001
+    name: determined-torvalds-b69001
 spec:
     color: '#fafafc'
     name: cardinality
 ---
 apiVersion: influxdata.com/v2alpha1
-kind: Variable
+kind: Task
 metadata:
-    name: amazing-boyd-93b003
+    name: unruffled-sinoussi-769001
 spec:
     associations:
       - kind: Label
-        name: kind-spence-53b001
-    language: flux
-    name: bucket
+        name: determined-torvalds-b69001
+    every: 1d
+    name: 'Cardinality Alert '
+    offset: 20m0s
     query: |-
+        import "influxdata/influxdb"
+        import "slack"
+
+
+
```

Comment:
add a header here that describes the variables and what the user has to do with them.